### PR TITLE
Update retinanet_swin-t-p4-w7_fpn_1x_coco.py

### PR DESCRIPTION
**Pull request by ShengliLiu**
Fix the error encountered during loading the pretrained backbone as follows,

2022-01-10 09:45:12,308 - mmdet - INFO - load checkpoint from http path: https://github.com/SwinTransformer/storage/releases/download/v1.0.0/swin_small_patch4_window7_224.pth
Traceback (most recent call last):
  File "tools/train.py", line 185, in <module>
    main()
  File "tools/train.py", line 159, in main
    model.init_weights()
  File "/home/shengliliu/anaconda3/envs/cots/lib/python3.7/site-packages/mmcv/runner/base_module.py", line 116, in init_weights
    m.init_weights()
  File "/home/shengliliu/anaconda3/envs/cots/lib/python3.7/site-packages/mmdet/models/backbones/swin.py", line 706, in init_weights
    if list(state_dict.keys())[0].startswith('module.'):
IndexError: list index out of range

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/swin/retinanet_swin-t-p4-w7_fpn_1x_coco.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
         # Please only add indices that would be used
         # in FPN, otherwise some parameter will not be used
         with_cp=False,
+        convert_weights=True,
         init_cfg=dict(type='Pretrained', checkpoint=pretrained)),
     neck=dict(in_channels=[192, 384, 768], start_level=0, num_outs=5))
```
Comment (Czm369) [inline configs/swin/retinanet_swin-t-p4-w7_fpn_1x_coco.py:29]: Thanks. The fix is right.

**Review by Czm369 — APPROVED** [review]
Thanks. The Fix is right.

**Comment by jshilong** [comment]
LGTM

**Comment by OpenMMLab-Assistant003** [comment]
Hi @ShengliLiu！First of all, we want to express our gratitude for your significant PR in the MMDetection project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤ @ShengliLiu

**Pull request merged**